pytest.ini

```diff
@@ -85,6 +85,4 @@ norecursedirs =
 
 testpaths = tests/
 
-pythonpath = src
```

Comment:
Should revert this from the `src` PR in a separate PR since this likely was not needed

Reply:
https://github.com/aio-libs/propcache/pull/27